website_logo/views/website_templates.xml

```diff
@@ -1,13 +1,13 @@
-<?xml version="1.0" encoding="utf-8"?>
 <odoo>
-
-    <template id="layout_logo_show" inherit_id="website.layout_logo_show">
-        <xpath expr="//a//span" position="attributes">
-            <attribute name="class" add="hide" separator=" "/>
+    <template id="add_logo_to_website" inherit_id="website.layout_logo_show" customize_show="False" name="Show Logo">
```

Comment:
Better `customize_show="True" active="True"`. This way, this default behavior is enabled, but can be disabled per website.

Just adding that should clear the task that you specified in `ROADMAP.rst`.